projects/igniteui-angular/src/lib/data-operations/filtering-condition.ts

```diff
@@ -548,7 +548,7 @@ export class IgxStringFilteringOperand extends IgxFilteringOperand {
      * @memberof IgxStringFilteringOperand
      */
     public static applyIgnoreCase(a: string, ignoreCase: boolean): string {
-        a = a || '';
+        a = a === null || a === undefined ? '' : a;
```

Comment:
Maybe use the [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)
```suggestion
        a = a ?? '';
```

Reply:
That's a good idea. Unfortunately we have already merged it that way in 11.0.x.

@mtsvyatkova, Please create a new PR for 11.0.x with this change and apply it to the other PRs.